classes/publication/maps/Schema.php

```diff
@@ -155,6 +155,9 @@ function ($citation) {
                 case 'fullTitle':
                     $output[$prop] = $publication->getFullTitles('html');
                     break;
+                case 'versionDataDisplay':
```

Comment:
I think `versionString` might be a little less awkward than `versionDataDisplay` (here and elsewhere).

Reply:
Changed to `versionString`